src/MainWindow.vala

```diff
@@ -1177,9 +1161,13 @@ namespace Terminal {
             // Closing a tab will switch to another, which will trigger check for same names
         }
 
-        private void action_new_tab () {
+        public void action_new_tab () {
```

Comment:
This doesn't need to be public.

Reply:
Oh yes, I was going to call it directly but opted for activate_action instead. Will revert.